bs4.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+   <link rel="stylesheet" href="https://rawgit.com/twbs/bootstrap/v4-dev/dist/css/bootstrap.css" />
+   <link rel="stylesheet" href="//netdna.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.css" />
+   <link rel="stylesheet" href="../dist/summernote-bs4.css">
```

Comment:
1. `<link>` is a void element. So you don't have to type ` />`.
http://www.w3.org/TR/html-markup/syntax.html#syntax-elements
2. be careful of indentation. after `<meta>`.